src/app.ts

```diff
@@ -1,4 +1,5 @@
-import path from 'path'
```

Comment:
I don't think you need to use the star notation here.  The tsconfig we are using has `allowSyntheticDefaultImports` set to true.